src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -578,7 +580,7 @@ function mapStateToProps({
             fetchingContributors ||
             fetchingContributorTypes ||
             fetchingCountries,
-        embed,
+        embed: !!embed,
```

Comment:
This worked previously because `embed` is set to `"1"`, which will evaluate to `true`. However, it is better to convert it to a boolean. We do this in other components where `embed` is passed as a prop.